Userland/Demos/VirGLDemo/CommandBufferBuilder.cpp

```diff
@@ -80,6 +80,13 @@ class CommandBuilder {
     bool m_finalized { false };
```

Comment:
This (last) commit seems like a candidate for squashing

Reply:
Makes sense.

Reply:
Will fix in next revision.